lang/en/tool_objectfs.php

```diff
@@ -162,11 +162,11 @@
 $string['settings:presignedcloudfronturl:cloudfront_key_pair_id'] = 'Key_Pair ID from AWS';
 $string['settings:presignedcloudfronturl:cloudfront_key_pair_id_help'] = 'This is generated using AWS account \'root\' user (along with the private key .pem file).';
 $string['settings:presignedcloudfronturl:cloudfront_private_key_pem'] = 'PRIVATE Key .pem';
-$string['settings:presignedcloudfronturl:cloudfront_private_key_pem_help'] = 'Private key content (.pem file).';
+$string['settings:presignedcloudfronturl:cloudfront_private_key_pem_help'] = 'Private key content (.pem file) or relative path to cloudfront.pem file. ($CFG->dataroot . "/cloudfront.pem")';
```

Comment:
This string seems inconsistent with the implementation?

Reply:
updated